github_connector/models/abstract_github_model.py

```diff
@@ -63,17 +63,31 @@ def github_login_field(self):
             return self._github_login_field
 
     @api.model
-    def get_odoo_data_from_github(self, data):
-        """Prepare function that map Github data to create in Odoo"""
+    def get_conversion_dict(self):
+        """
+        Prepare function that map Github fields to Odoo fields
+        :return: Dictionary {odoo_field: github_field}
+        """
+        """"""
```

Comment:
pylint error:
github_connector/models/abstract_github_model.py:71: [W0105(pointless-string-statement), AbstractGithubModel.get_conversion_dict] 

Just delete this unused string statement

Reply:
done